core.fnl

```diff
@@ -1,72 +1,122 @@
 (hs.console.clearConsole)
```

Comment:
Extended documentation of the loading cycle and initialization of modules

Reply:
Done in #36